python/setup.py

```diff
@@ -336,6 +336,7 @@ def get_packages(self):
         "aiosignal",
         "frozenlist",
         "requests",
+        "watchfiles",
```

Comment:
@ericl @richardliaw can one of you stamp for this please? I think mine doesn't count because I pushed a commit to the branch.

Reply:
should this be in `serve` instead of a core dependency?

Reply:
https://github.com/ray-project/ray/pull/38389/files#diff-eb8b42d9346d0a5d371facf21a8bfa2d16fb49e213ae7c21f03863accebe0fcfL272 should go here instead

Reply:
I have moved the `watchfiles` dependency to the `serve` dependencies.